synapse/storage/background_updates.py

```diff
@@ -131,7 +133,7 @@ def start_doing_background_updates(self):
             return True
 
         # obviously, if we have things in our queue, we're not done.
```

Comment:
Could you update that comment please? We're not dealing with a queue anymore.

Reply:
done